inst/@sym/limit.m

```diff
@@ -83,6 +83,10 @@
       print_usage ();
   end
 
+  if (isempty (x))
+    x = sym('x');
```

Comment:
if x is empty, can we just return f?, without a limit var, the limit will be the same function.